src/main/scripts/org/reaktivity/specification/amqp/connection/open.exchange/client.rpt

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright 2016-2019 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect await ROUTED_SERVER
+        "nukleus://streams/amqp#0"
+  option nukleus:window 8192
+  option nukleus:transmission "duplex"
+
+connected
+
+# header exchange
+write "AMQP" [0x00 0x01 0x00 0x00]
+read "AMQP" [0x00 0x01 0x00 0x00]
+
+# open frame exchange
+write [0x00 0x00 0x00 0x18]                                      # length = 24
+      [0x02]                                                     # dof = 2
+      [0x00]                                                     # type = AMQP (0)
+      [0x00 0x00]                                                # channel = 0
+      [0x00 0x53 0x10]                                           # performative = open (16)
+      [0xd0 0x00 0x00 0x00 0x08]                                 # list type (list32), physical length = 8
+      [0x00 0x00 0x00 0x01]                                      # number of fields in list = 1
+      [0xa1 0x02 0x63 0x31]                                      # type = string, length = 2, container-id = c1
+
+read [0x00 0x00 0x00 0x19]                                       # length = 25
+     [0x02]                                                      # dof = 2
+     [0x00]                                                      # type = AMQP (0)
+     [0x00 0x00]                                                 # channel = 0
+     [0x00 0x53 0x10]                                            # performative = open (16)
+     [0xc0 0x0c]                                                 # list type (list8), physical length = 12
+     [0x01]                                                      # number of fields in list = 1
+     [0xa1 0x09 0x6c 0x6f 0x63 0x61 0x6c 0x68 0x6f 0x73 0x74]    # type = string, length = 9, container-id = localhost
```

Comment:
Recommend changing command to `# container-id = "localhost"
Please make changes consistently across all similar files.